README.md

```diff
@@ -74,6 +74,19 @@ docker run -d \
   briefercloud/briefer
 ```
 
+When running on Windows' PowerShell, it might be necessary to add an extra to the end of each line instead of the `\`, like this:
```

Comment:
Hey @Cauatn, I think your editor cut out the ` character here:

```suggestion
When running on Windows' PowerShell, it might be necessary to add an extra ` to the end of each line instead of the `\`, like this:
```